x86_64-options.mk

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2017-2018 Intel Corporation
```

Comment:
I think this should probably be copyright ARM as you created it.

I'm in two minds about having this file though. At this stage, I think it would be better if this file did not exist and the variables are defined in `Makefile` for two reasons:

1) By having these variables in the main `Makefile`, all architectures by default will be using the default values.

    If we add support for further architectures and those architectures can use the same values as x86_64, we'll have duplication in `x86_64-options.mk` and `${new_arch}-options.mk`.

2) It introduces a maintenance issue.

    The `UNION` variable lists all the targets that are defined in `Makefile`. I think it would be better to have `UNION` in `Makefile` where the targets are defined to make it easier to remember to update that variable as new test types are added.

Reply:
... but to be clear we will still need a `aarch64-options.mk` :)

Reply:
ok. I will relocate them to Makefile as default values for all archs. and reload specific value in aarch64-options.mk to list currently supported integration tests in arm64